index.html

```diff
@@ -272,6 +272,16 @@ <h3>
       The <dfn data-dfn-for="DigitalCredential">data</dfn> member is the
       credential's response data.
     </p>
+    <h3>
+      [[\DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)
+      internal method
```

Comment:
WebAuthn [seems to say](https://w3c.github.io/webauthn/#sctn-permissions-policy) we can't do this here due to lack of access to the settings object. I haven't looked into the details though, WDYT?

Reply:
Ah, right! I need to add it to the Cred Man itself! That's what they do for WebAuthn. I'll do that.

Reply:
@RByers sent https://github.com/w3c/webappsec-credential-management/pull/242